src/protocol/ldap/org/apache/jmeter/protocol/ldap/sampler/LDAPExtSampler.java

```diff
@@ -85,6 +85,8 @@
     public static final String PORT = "port"; // $NON-NLS-1$
 
     public static final String SECURE = "secure"; // $NON-NLS-1$
+    
+    public static final String TRUSTALL = "TrustAll";
```

Comment:
The other key values are all lowercase. Any special reason for CamelCase?

Reply:
no, just working with command line tools that have that explicit option have put me in the habit of spelling it that way. I will update it.